cumulus/parachains/runtimes/assets/asset-hub-westend/src/bridge_to_ethereum_config.rs

```diff
@@ -127,9 +158,16 @@ where
 		if !IsForeign::contains(asset_location, &origin_location) {
 			return Err(origin)
 		}
+		let asset_location: Location = asset_location.clone().into();
+		let owner = AssetInspect::owner(asset_location.into());
+		let location: Location = origin_location.clone().into();
+		let from = LocationToAccountId::convert_location(&location);
+		if !owner.eq(&from) {
+			return Err(origin)
+		}
 		let latest_location: Location =
 			origin_location.clone().try_into().map_err(|_| origin.clone())?;
```

Comment:
There is already `location: Location` variable available
```suggestion
```

Reply:
https://github.com/vgeddes/polkadot-sdk/pull/57/commits/6e1aa1c8b07b49e938d1b2058d611bd26f9dcfb7